tests/core-addon/selenium/core_addon.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public

```

Comment:
Instead of calling the directory `selenium`, can we rather call it `integration`? It might be beneficial to also move the unit tests to a `unit` directory.